_saas-integrations/closeio/v1/closeio-v1.md

```diff
@@ -0,0 +1,90 @@
+---
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: closeio
+permalink: /integrations/saas/closeio
+tags: [saas_integrations]
+keywords: closeio, integration, schema, etl closeio, closeio etl, closeio schema
+layout: singer
+# input: false
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "closeio"
+display_name: "Close.io"
+
+singer: true
+tap-name: ""
+repo-url: https://github.com/singer-io/tap-closeio
+
+# this-version: ""
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+status: "Alpha"
+certified: true
+
+historical: "1 year"
+frequency: "30 minutes"
+tier: "Paid"
+status-url: "http://status.close.io/"
+icon: /images/integrations/icons/closeio.svg
+whitelist:
+  tables: false
+  columns: false
+
+anchor-scheduling: true
+extraction-logs: true
+loading-reports: true
+
+# attribution-window: "# days"
+# attribution-is-configurable:
+
+# setup-name: ""
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**A Close.io API Key**"
+
+requirements-info: |
```

Comment:
This should go under the `setup-steps` section with the same title. Right now that's line 69. [Here's the metadocs guide for it](https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/#content-sections).

For example:

```
setup-steps:
  - title: "Generate a Close.io API Key"
    anchor: "generate-closeio-api-key"
    content: |
      [steps you wrote up there go here]
```